src/unicorn/qubot.rs

```diff
@@ -377,27 +412,16 @@ impl<'a> Qubot<'a> {
 
         writeln!(out)?;
 
-        // TODO: If Qubits are always referenced by their `name: u64` this can
-        // be simplified by just typedef'ing `Qubit` instead.
-        let mut sorted_linear_coeffs: Vec<(&HashableQubitRef, &i32)> =
-            self.qubo.linear_coefficients.iter().collect();
-        sorted_linear_coeffs.sort_by_key(|(qubit, _)| (*qubit.value.borrow()).name);
-        for (qubit, coeff) in sorted_linear_coeffs {
+        for (qubit, coeff) in self.qubo.linear_coefficients.iter() {
```

Comment:
This undoes my fix that sorts coefficients by the Qubit number. Can we restore that? I think it is still needed.